server/pkg/schema/builder.go

```diff
@@ -55,3 +55,8 @@ func (b *Builder) Fields(fields FieldList) *Builder {
 	b.s.fields = fields.Clone()
 	return b
 }
+
+func (b *Builder) TitleField(fId *FieldID) *Builder {
+	b.s.titleField = fId.CloneRef()
```

Comment:
```suggestion
func (b *Builder) TitleField(fid *FieldID) *Builder {
	b.s.titleField = fid.CloneRef()
```

Also you forgot titleField validation in the `Build` method.

Reply:
TitleField is optional, what kind of validation is required in the build method?

Reply:
Done!